README.rst

```diff
@@ -60,41 +60,194 @@ See the collectd `config guide <https://collectd.org/documentation/manpages/coll
 collectd_plugin_capabilities_*
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 These vars are ones passed to the ``capabilities`` plugin.
-See the collectd `wiki <https://collectd.org/wiki/index.php/Plugin:capabilities>`_ for details.
-collectd_plugin_capabilities_host: localhost
-collectd_plugin_capabilities_port: "9104"
+See the collectd `config guide <https://collectd.org/documentation/manpages/collectd.conf.5.shtml#plugin_capabilities>`_ for details.
```

Comment:
One of the things I saw on @csibbitt upstream review was that he was told to delete the README.md file because it is auto generated. Chris, can you confirm?

Reply:
@leifmadsen, that is probably the tripleo-qdr-config-role that you're thinking of, which will be going into tripleo_ansible. The equivalent here is tripleo_collectd_config_role.
This role will remain standalone.

Reply:
Ahhhh got it. Thanks for the info!